docs/en/quick-answers/developing/setting_software_environment_variables.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: How do I set environment variables before launching software?
+pagename: setting_software_environment_variables
+lang: en
+---
+
+# How do I set environment variables before launching software?
+
+It's common to need to set environment before launching a given software so that your studio customizations are applied.
```

Comment:
Not sure if you meant to add "variables" between "environment" and "before". Makes sense either way...

Reply:
Sabrina suggested removing the whole sentence so I'll do that instead, but you're right it doesn't read well.